src/adhocracy_core/adhocracy_core/sheets/relation.py

```diff
@@ -0,0 +1,122 @@
+"""Sheets for statements about relations between process content/comments."""
+import colander
+
+from adhocracy_core.interfaces import IPredicateSheet
+from adhocracy_core.interfaces import ISheet
+from adhocracy_core.interfaces import ISheetReferenceAutoUpdateMarker
+from adhocracy_core.interfaces import SheetToSheet
+from adhocracy_core.schema import AdhocracySchemaNode
+from adhocracy_core.schema import PostPool
+from adhocracy_core.schema import Reference as ReferenceSchema
+from adhocracy_core.schema import UniqueReferences
+from adhocracy_core.schema import Reference
+from adhocracy_core.sheets import add_sheet_to_registry
+from adhocracy_core.sheets import sheet_meta
+
+
+class IPolarization(IPredicateSheet, ISheetReferenceAutoUpdateMarker):
+
+    """Marker interface for the polarization sheet."""
+
+
+class IPolarizable(ISheet, ISheetReferenceAutoUpdateMarker):
+
+    """Marker interface for resources than can be polarized."""
+
+
+class ICanPolarize(ISheet):
+
+    """Marker interface for resources that can polarize."""
+
+
+class Position(AdhocracySchemaNode):
+
+    """Schema node for the side (pro or contra)."""
+
+    schema_type = colander.String
+    missing = colander.drop
+    default = 'pro'
+    validator = colander.OneOf(['pro', 'contra']),
+
+
+class PolarizationSubjectReference(SheetToSheet):
+
+    """Reference from polarization to polarize."""
+
+    source_isheet = IPolarization
+    source_isheet_field = 'subject'
+    target_isheet = ICanPolarize
+
+
+class PolarizationObjectReference(SheetToSheet):
+
+    """Reference from polarization to polarized resource."""
+
+    source_isheet = IPolarization
+    source_isheet_field = 'object'
+    target_isheet = IPolarizable
+
+
+class PolarizationSchema(colander.MappingSchema):
+
+    """Polarizable sheet data structure.
+
+    `position`: the position in the debate, 'pro' or 'contra'.
+    """
+
+    subject = ReferenceSchema(reftype=PolarizationSubjectReference)
+    object = ReferenceSchema(reftype=PolarizationObjectReference)
+    position = Position()
+
+polarization_meta = sheet_meta._replace(isheet=IPolarization,
+                                        schema_class=PolarizationSchema,
+                                        create_mandatory=True)
```

Comment:
we should set implicit_addable to True, otherwise we cannot add the polarization to the relations service pool.